pkg/monitortests/kubeapiserver/disruptionnewapiserver/monitortest.go

```diff
@@ -32,7 +33,13 @@ func (w *newAPIServerDisruptionChecker) CollectData(ctx context.Context, storage
 	if err != nil {
 		return nil, nil, err
 	}
-
+	isMicroShift, err := exutil.IsMicroShiftCluster(kubeClient)
+	if err != nil {
+		return nil, nil, err
+	}
+	if isMicroShift {
+		return nil, nil, nil
```

Comment:
Wondering if there is value to carrying over the notSupportedReason like in https://github.com/pacevedom/origin/blob/ecdb4fbd1761c42b0eab8c0e41a4085aaa1ebb14/pkg/monitortests/network/disruptionserviceloadbalancer/monitortest.go#L84 

and then a check for it in https://github.com/pacevedom/origin/blob/ecdb4fbd1761c42b0eab8c0e41a4085aaa1ebb14/pkg/monitortests/kubeapiserver/disruptionnewapiserver/monitortest.go#L69

Reply:
Done